info.go

```diff
@@ -182,3 +182,19 @@ func scanFdInfoReader(r io.Reader, fields map[string]interface{}) error {
 
 	return nil
 }
+
+// EnableStats starts the measuring of the runtime (in ns)
+// and run counts.
+//
+// Requires at least 5.8.
+func EnableStats() error {
```

Comment:
At the moment only `BPF_STATS_RUN_TIME` is supported to be set as value in the attribute. Maybe (I don't know) the kernel will provide more `StatsTyp` at some point.
Do we want to change the API then or ask now the user to provide an argument to `EnableStats()`?

Reply:
I'd go with an extra parameter since the kernel API encourages extension. I wonder whether it should be `(which int)` or `(which StatsType)`.

Reply:
I added the extra argument for the function.